SPECS/clang/clang.spec

```diff
@@ -120,20 +120,10 @@ A set of extra tools built using Clang's tooling API.
 %prep
```

Comment:
Did we miss to bump the release for clang ?

Reply:
Nope, we just haven't built this one even once, yet, so there's no need.